.github/workflows/builds.yml

```
@@ -7,12 +7,21 @@ name: Build
 jobs:
   build:
     name: Build contracts
-    runs-on: k8s-worker-8
+    runs-on: github-hosted-heavy-runner
     steps:
       - name: Clone the repository
         uses: actions/checkout@v4
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)